### ISSUE TITLE
vscode-eslint error when running in mono repo packages

### ISSUE THREAD

**Issue by jaydenseric** [apollographql/eslint-plugin-graphql#105]
With this repo/project structure:

* api/
* app/
  * components/
  * schema.graphql
  * .graphqlconfig (with `"schemaPath": "schema.graphql"`)
  * package.json (with `eslintConfig` and `eslint-plugin-graphql` configured)

[`vscode-eslint`](https://github.com/Microsoft/vscode-eslint) displays this error when attempting to lint files in `/app/components` containing GraphQL syntax:

<img width="1568" alt="screen shot 2018-01-22 at 3 40 02 pm" src="https://user-images.githubusercontent.com/1754873/35205728-9e7721b8-ff8b-11e7-88f9-04254992aec2.png">

The error (which BTW in a narrow window cuts off with an ellipses without being able scroll) suggests looking at the ESLint output, which doesn't show any errors:

<img width="827" alt="screen shot 2018-01-22 at 3 40 42 pm" src="https://user-images.githubusercontent.com/1754873/35205804-2358a262-ff8c-11e7-8286-2ba9462310d4.png">

Interestingly, moving `.graphqlconfig` and `schema.graphql` out into the root of the project solves the errors. This is not a solution though.

Adding `./` to the `schemaPath` in `app/.graphqlconfig` has no effect.

Running `eslint .` in Terminal within `/app` works ok.

**Comment by jnwng** [comment]
Thanks - I noticed this recently too. We need to use a different function
from ‘GraphQL-config’ but unfortunately they don’t transpire yet and it
would affect our compat matrix. I’m working on getting those upstream
dependencies compatible and that should get us the behavior we expect

**Comment by jaydenseric** [comment]
Looks like `graphql-config` [does not have a package.json `engines` field](https://github.com/graphcool/graphql-config/blob/master/package.json). I can't easily tell what versions of Node.js are supported.

**Comment by jnwng** [comment]
The function we use is compatible back to Node 4 but the one we want
(findconfigforfile or something) is Node 8

**Comment by jaydenseric** [comment]
As a really gross temporary workaround, you can create symbolic links in the project root:

```shell
ln -s app/.graphqlconfig .graphqlconfig
ln -s app/schema.graphql schema.graphql
```

**Comment by jnwng** [comment]
this affects atom too.

see https://github.com/graphcool/graphql-config/pull/84 and https://github.com/graphcool/graphql-import/pull/63 which should allow us to use `getConfigForFile` here in in the supported Node range of this library.
